client/src/components/Typeahead/Typeahead.tsx

```diff
@@ -54,19 +64,14 @@ interface TypeaheadState {
   results?: ResultProps[];
 }
 
-export class Typeahead extends React.Component<
-  TypeaheadProps,
-  TypeaheadState,
-  {}
-> {
+export class Typeahead extends React.Component<TypeaheadProps, TypeaheadState> {
   menu_id: string;
 
   private typeahead_ref = React.createRef<HTMLDivElement>();
-  private virtualized_list_ref = React.createRef<any>();
+  private virtualized_list_ref = React.createRef<List>();
 
   static defaultProps = {
     placeholder: text_maker("search"),
-    min_length: 3,
```

Comment:
I think this default's still valid. Let me know if I'm wrong about that!

Reply:
oops didn't mean to delete that one